docs/book/src/multiversion-tutorial/deployment.md

```diff
@@ -119,3 +119,5 @@ resource.version.group` is for everything else.
 [steps for troubleshooting](/TODO.md)
 
 [ref-multiver]: /reference/generating-crd.md#multiple-versions "Generating CRDs: Multiple Versions"
+
+[crd-version-pref]: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-priority "Versions in CustomResourceDefinitions"
```

Comment:
Why do we have "Versions in CustomResourceDefinitions" in the alias for the link?
```suggestion
[crd-version-pref]: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-priority 
```

Reply:
I referred this URL and alias from [other PR](https://github.com/kubernetes-sigs/kubebuilder/pull/2384/files#diff-5450fdfc6c1e06b3e8f71ecd3b3025734f6098366820c52844672587c9d194abL123).
This alias text will showed as popup when hover the cusor to this link.

Other link "ref-multiver" is also same manner, it have alias text, so it's good that alias text is set, I think.

Reply:
Thank you for the clarifications